scripts/lint.py

```diff
@@ -30,7 +30,7 @@
 
 any_todo_pattern = re.compile(r"TODO\(.*\)")
 legal_todo_inner_pattern = re.compile(
-    r"TODO\(((?:[a-zA-Z\-_/]+)?#\d+|[a-zA-Z]+)(?:,\s*((?:[a-zA-Z\-_/]+)?#\d+|[a-zA-Z]+))*\)"
+    r"TODO\(((?:[a-zA-Z\-_/]+[a-zA-Z\-_/\d]+)?#\d+|[a-zA-Z]+)(?:,\s*((?:[a-zA-Z\-_/]+)?#\d+|[a-zA-Z]+))*\)"
```

Comment:
this is over-complicated. so much unnecessary stuff in there

`TODO\([\w-]*(#\d+)?\):?`

should suffice

Reply:
... I gonna ignore this for now and merge ahead. Users deserve this sample working on windows, independent of this regex 😄

Reply:
should be fixed in https://github.com/Pix4D/pyopf/pull/9 ;) thanks